### DISCUSSION TITLE
How can we get start and end point of a entity

### DISCUSSION THREAD

**Discussion by aditiaryamantechnologies** [mozman/ezdxf#924] [Issues]
Hi we are trying to read dxf file and create polygon using that. Our file contains nested objects (polygon, arcs, etc) and we are not able to determine which line belongs to which polygon. Can you assist.. We have attached our dxf file for reference.
[11001.zip](https://github.com/mozman/ezdxf/files/12437924/11001.zip)

**Comment by mozman** [comment]
Shape recognition is not the domain of `ezdxf'`, it's a DXF interface to read/write/modify DXF files, everything beyond that is your job.
